### PR TITLE
More balls

### PR DESCRIPTION
**Pull request by carlfjones**
If the player manages to get past the first shark - the ship fires more cannonballs.
I'll be honest it is mayhem.

### CODE REVIEW

File: unity_game/Assets/Scripts/DragonCollision.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class DragonCollision : MonoBehaviour
```
Comment (Danielandro) [inline unity_game/Assets/Scripts/DragonCollision.cs:5]: Glad the trigger worked

File: unity_game/Assets/Scripts/SharkMovement.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SharkMovement : MonoBehaviour
+{
+    // Start is called before the first frame update
+    public ParticleSystem explosion;
+    [SerializeField]
+    GameObject arrow;
```
Comment (Danielandro) [inline unity_game/Assets/Scripts/SharkMovement.cs:10]: really need to change the name of some of these variables

File: unity_game/ProjectSettings/TagManager.asset [hunk 3]
```diff
@@ -3,7 +3,8 @@
 --- !u!78 &1
 TagManager:
   serializedVersion: 2
-  tags: []
+  tags:
+  - Dragon
```
Comment (Danielandro) [inline unity_game/ProjectSettings/TagManager.asset:7]: Tag, You're it!

**Review by Danielandro — APPROVED** [review]
Other than the merge conflicts - which are gitignore related again - this looks solid.
Need to rename some of the variables so their purpose is clearer.

**Review by chris-groves — APPROVED** [review]

**Pull request merged**